pretty_gpx/gpx/gpx_track.py

```diff
@@ -87,7 +87,15 @@ def load(gpx_path: str | bytes) -> tuple['GpxTrack', float, float]:
             plt.ylabel('Elevation (in m)')
             plt.show()
 
-        return gpx_track, gpx.length_3d()*1e-3, gpx.get_uphill_downhill().uphill
+        print(all_segment_cumul_d,gpx.length_3d()*1e-3)
+        print(all_segment_cumul_ele,gpx.get_uphill_downhill().uphill)
```

Comment:
make the log more readable :art:

Reply:
Firstly I tought to remove it but now I've added text, is there a common DEBUG variable ?